pyproject.toml

```diff
@@ -143,6 +143,7 @@ isort.known-first-party = ["annubes"]
     "S101",   # Use of `assert` detected
     "ANN201", # Missing return type
     "D103",   # Missing function docstring
-    "SLF001", # private member access
+    "SLF001", # Private member access
+    "ANN401", # Function arguments annotated with too generic `Any` type
```

Comment:
I'm not a huge fan of this. I get that it's not pretty to turn it off many times over on individual functions, but I do think it's a good practice to avoid using `Any` in most cases.

Alternatively, at the top of the test file in question, you can state `AnyType = Any  #noqa: ANN401, RUF100` as a type alias and then use AnyType wherever you don't want the linter to check without having to noqa each instance.
(the RUF100 ignoring isn't strictly necessary right now, but that is due to a bug in RUF, which I assume will be fixed at some point and then we won't have to fix it post-hoc).

Reply:
In the tests, you do want to test Any types. In the code-base I agree, that wouldn't be good. Very likely, we will need to state the exception at the beginning of each test file, so in my opinion is better to just set it in the toml directly on all tests files. This won't disable the error in the code-base, of course.